client/src/components/Notices/CompaniesTable.scss

```diff
@@ -1,20 +1,20 @@
 @import '../variables.scss';
 
 .companies-table {
-  background-color: white;
+  background-color: $component-background;
   padding: 0 0 0 20px;
   border-radius: 1px;
   margin-bottom: 0;
 }
 
 .companies-table-header tr th {
-  color: #6c8294; // TODO: Move to variables
+  color: $secondary;
```

Comment:
:+1:

Reply:
sorry but no

don't create new color variables, there's plenty of colors defined already - one of the existing should fit the purpose well

even if you really must define a new one and don't see other way around it, don't call a color used exactly once secondary